lib/hammer_cli_foreman_ansible/job_template.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module HammerCLIForemanAnsible
+  class JobTemplate < HammerCLIForemanRemoteExecution::JobTemplate
+    resource :job_templates
+
+    class InfoCommand < HammerCLIForemanRemoteExecution::JobTemplate::InfoCommand
+      build_options
+
+      extend_with(HammerCLIForemanAnsible::CommandExtensions::JobTemplate.new)
+    end
+
+    autoload_subcommands
+  end
+end
```

Comment:
Couldn't we just extend the original `HammerCLIForemanRemoteExecution::JobTemplate::InfoCommand` class directly?

Reply:
We can. I'll update it.